docs/src/development/sanitize-db.md

```diff
@@ -0,0 +1,143 @@
+The `log4j` security vulnerability is an example of what missing input sanitization can cause.
+
+NoSQL Databases ([MongoDB](../configuration/services/mongodb/_index.md), ...) can benefit from sanitization but for the sake of simplicity, 
+these use cases won't be covered in this guide.
+
+You'll need:
+
+- a project that uses a [MySQL database](../configuration/services/mysql/_index.md),
+- the [Platform CLI](/development/cli/_index.md#cli-command-line-interface) installed locally.
+
+## Sanitize the Database
+
+Make sure that the sanitization only runs on the development environments and **never** on the production environment.
+Otherwise you will loose most — if not all — of the relevant data stored in your database.
+
+{{< codetabs >}}
+
+---
+title=Manually
+file=none
+highlight=false
+---
+
+Assumptions:
+
+- `staging` is the name of the database that you want to sanitize.
+- The `staging` database is an exact copy of your production database.
+- `user` is the table where all your PII is stored.
+
+1. As a safety precaution, take a [database dump](../configuration/services/mysql.md#exporting-data). Run `platform db:dump -e staging` to get a database dump of your staging environment,
+1. Connect to the `staging` database with `platform sql -e staging`,
+1. Display all fields from your `user` database with `SELECT * FROM users;`.
+   Which could result in something along the lines of:
+
+   ```sql
+   MariaDB [main]> SELECT * FROM users;
+   +----+------------+---------------+---------------------------+---------------+
+   | ID | first_name | last_name     | user_email                | display_name  |
+   +----+------------+---------------+---------------------------+---------------+
+   |  1 | admin      | admin         | admin@yourcompany.com     | admin         |
+   |  2 | john       | doe           | john.doe@gmail.com        | john          |
+   |  3 | jane       | doe           | janedoe@ymail.com         | jane          |
+   +----+------------+---------------+---------------------------+---------------+
+   3 rows in set (0.00 sec)
+   ```
+
+1. Change the fields where PII Data is contained with [`UPDATE` Queries](https://dev.mysql.com/doc/refman/8.0/en/update.html).
+   For example, changing the first name of an user, when the email address is not using the company domain, would result in the following query:
+
+   ```sql
+   UPDATE user
+   SET first_name='redacted'
+   WHERE email NOT LIKE '%@yourcompany%'
+   ```
+
+   Adapt and run that query for all fields that need to be sanitized.
+   If you modified fields that shouldn't have been altered, [you can restore them from the previously taken dump](../administration/backup-and-restore.md#restore).
+
+   You can create a script to automate the sanitization process to be able to run it in [your deployment hook](../user_guide/reference/platform-app-yaml.html#hooks):
```

Comment:
I'm not sure where that link should go.

Also, given the `:`, tI would break this into two sentences or at least ideas. One about creating the script and the second about adding it to your `deploy` hook.